### PR TITLE
feat(rebuild-amplify): add new script

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
Govt masthead icons are kinda broken; the fix is out but we need to rebuild the apps in order to get the icons updatd

## Solution
- Get a list of all the apps
- For each app, get the latest job 
- Retry the latest job

## Notes
Logs are also attached for safety :monkas:

(can use[ shaz site](https://ogp-shazli-staging.netlify.app/resource-room/concerts/) for testing) 
This has not been tested yet; please test with the below steps: 
- [ ] go to a site with broken govt masthead icons (most sites, as long as not top level page) 
- [ ] run the script for that one site 
- [ ] check to see that site is rebuilt w/ working icons

### CODE REVIEW

File: rebuild-amplify/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (harishv7) [inline rebuild-amplify/package.json:1]: what is the master.txt? I dont think we need that?

File: rebuild-amplify/rebuild.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```
Comment (harishv7) [inline rebuild-amplify/rebuild.sh:1]: can we add comments this is an alternative script to TS version

File: rebuild-amplify/rebuild-amplify.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
+import { createObjectCsvWriter } from "csv-writer";
```
Comment (harishv7) [inline rebuild-amplify/rebuild-amplify.ts:1]: maybe add comments of the pre-setup req. needs aws creds to be setup

File: rebuild-amplify/package.json (hunk 1 above)
Comment (harishv7) [reply rebuild-amplify/package.json:1]: same for the other text files

Comment (seaerchin) [reply rebuild-amplify/package.json:1]: see description - i added the logs in as a safety measure. i can remove if you feel this is extra HAHA (cos we ran migration already)

File: rebuild-amplify/rebuild.sh (hunk 2 above)
Comment (seaerchin) [reply rebuild-amplify/rebuild.sh:1]: sgtm

File: rebuild-amplify/rebuild-amplify.ts (hunk 3 above)
Comment (seaerchin) [reply rebuild-amplify/rebuild-amplify.ts:1]: i think that's mostly true of this repo as a whole and not just this file in particular. maybe we should delegate this to another PR (iirc we have this in notion)

Comment (harishv7) [reply rebuild-amplify/rebuild-amplify.ts:1]: Yea haha we definitely should

Maybe for purpose here, can add 1-2 lines comments first?

File: rebuild-amplify/package.json (hunk 1 above)
Comment (harishv7) [reply rebuild-amplify/package.json:1]: I don't think we should be committing the raw logs with the IDs etc.

Comment (seaerchin) [reply rebuild-amplify/package.json:1]: deleted

File: rebuild-amplify/rebuild-amplify.ts (hunk 3 above)
Comment (seaerchin) [reply rebuild-amplify/rebuild-amplify.ts:1]: created a readme file

**Review by harishv7 — APPROVED** [review]

**Pull request merged**